letsencrypt/client/init.sls

```diff
@@ -13,6 +13,7 @@ certbot_packages:
     - names: {{ client.source.pkgs }}
     - watch_in:
       - cmd: certbot_installed
+    - fromrepo: jessie-backports
```

Comment:
You should have something like this in your `/etc/apt/preferences.d` to also handle upgrades correctly, etc.:
```
$ cat /etc/apt/preferences.d/distribution 
Package: *
Pin: release a=stretch
Pin-Priority: 500

Package: *
Pin: release a=stretch-backports
Pin-Priority: 500

Package: *
Pin: release a=testing
Pin-Priority: 400

Package: *
Pin: release a=unstable
Pin-Priority: 100

Package: *
Pin: release a=experimental
Pin-Priority: 50
```

Because using fromrepo is really a bad approach and useful barely only for initial installation.